Directory.Packages.props

```diff
@@ -84,7 +84,7 @@
     <PackageVersion Include="Polly" Version="8.4.2" />
     <PackageVersion Include="RabbitMQ.Client" Version="6.8.1" />
     <PackageVersion Include="Raven.CodeAnalysis" Version="1.0.11" />
-    <PackageVersion Include="Raven.Pal" Version="6.2.22" />
+    <PackageVersion Include="Raven.Pal" Version="6.2.41" />
```

Comment:
Shouldn't it be 7.0.x?

Reply:
Yes, but we'll probably want to run that through the numbers - and se it to 7.0 only when we are ready to release.
Because each change consume a cache entry in the NuGet cache, which we cannot easily reuse.

Reply:
Makes sense